ansible_collections/arista/avd/roles/eos_designs/docs/input-variables.md

```diff
@@ -1350,14 +1350,22 @@ Management interface is modified for specific platforms like modular platforms w
 roles/eos_designs/docs/tables/platform-settings.md
 --8<--
 
-## PTP settings
+## PTP
 
 See the [Configuring PTP](how-to/ptp.md) how-to for details.
 
 --8<--
 roles/eos_designs/docs/tables/ptp.md
 --8<--
 
+## PTP settings
+
+See the [Configuring PTP](how-to/ptp_settings.md) how-to for details.
+
+--8<--
+roles/eos_designs/docs/tables/ptp_settings.md
+--8<--
+
```

Comment:
I think a single section is enough here and then combining all the keys in the single table.

```suggestion
## PTP settings

See the [Configuring PTP](how-to/ptp.md) how-to for details.

--8<--
roles/eos_designs/docs/tables/ptp_settings.md
--8<--

```